moths/model.py

```
@@ -0,0 +1,28 @@
+import logging
+from dataclasses import dataclass
+
+from omegaconf import DictConfig
+from torch import nn
+from torchvision import models
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class ModelConfig(DictConfig):
+    zoo_name: str = "resnet50"
```

Why define defaults here if we use something like Hydra? I can imagine this default could lead to silent errors later on.

---

from when I started, will remove